src/main/java/Torchie.java

```diff
@@ -0,0 +1,79 @@
+import java.util.Arrays;
+import java.util.Scanner;
+public class Torchie {
+
+    private static Task[] taskStore = new Task[100];
+    private static int taskStoreSpace = 0; // keep track of next null space in array
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("Hello! I'm Torchie!");
+        System.out.println("What can I do for you?");
+        System.out.println("Let's play storetorchie today! You say something and I ll store it!");
+
+        String input;
+        do {
+            input = scanner.nextLine();
+            String firstWord = input.split(" ")[0];
+
+            switch(firstWord) {
+            case "list":
+                showList(taskStore);
+                break;
+            case "mark":
+                taskStore = markItem(taskStore, Integer.parseInt(input.split(" ")[1]));
```

Comment:
```suggestion
                int itemNumber = Integer.parseInt(input.split(" ")[1]);
                taskStore = markItem(taskStore, itemNumber);
```
Can dumb down for reader, add extra line to store the integer.